### PR TITLE
Docs/411 task logs

### PR DESCRIPTION
**Pull request by jstirnaman**
- Retrieve all logs for a task (closes https://github.com/influxdata/openapi/issues/411).
- Cancels a running task (closes https://github.com/influxdata/openapi/issues/420).
- Clarify BadRequestError example summary for org and orgID.

### CODE REVIEW

File: src/common/paths/tasks_taskID_runs_runID.yml [hunk 1]
```diff
@@ -19,23 +23,61 @@ get:
...
   responses:
     "200":
-      description: Returns the run record.
+      description: Success. The response body contains the task run.
       content:
         application/json:
           schema:
             $ref: "../schemas/Run.yml"
+        examples:
+          runSuccess:
```
Comment (glinton) [inline src/common/paths/tasks_taskID_runs_runID.yml:32]: I think this all just needs indented one more stop then re-run `make generate` to fix the openapi spec errors

File: src/common/responses/BadRequestError.yml [hunk 2]
```diff
@@ -5,14 +5,14 @@
...
+    - Returns this error if an incorrect value is passed for `org` or `orgID`.
   content:
     application/json:
       schema:
         $ref: "../schemas/Error.yml"
       examples:
-        orgNotFound:
-          summary: Organization not found
+        orgProvidedNotFound:
+          summary: The Org or orgID passed in the request doesn't match the token organization
```
Comment (sunbryely-influxdata) [inline src/common/responses/BadRequestError.yml:15]:
```suggestion
          summary: The org or orgID passed in the request doesn't match the token organization
```

**Comment by sunbryely-influxdata** [comment]
can we provide x-codeSamples for these?

**Comment by jstirnaman** [comment]
> can we provide x-codeSamples for these?

I can. I'm only adding code samples when the request (params) might be confusing. Whatever we replace Redoc with (hopefully in Q4) will generate samples from the spec.

**Review by glinton — APPROVED** [review]

**Pull request merged**